<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FormTelalogin</class>
 <widget class="QWidget" name="FormTelalogin">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>682</width>
    <height>640</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Login</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../assets/file.qrc">
    <normaloff>:/telainicial/telainicial/pdv.png</normaloff>:/telainicial/telainicial/pdv.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">QWiget#FormTelalogin{
	background-color: rgb(0, 0, 0,200);
}</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_2">
   <item>
    <widget class="QWidget" name="widget_2" native="true">
     <property name="styleSheet">
      <string notr="true">QWiget#widget_2{
	background-color: rgb(0, 0, 0,200);
}</string>
     </property>
     <widget class="QWidget" name="widget" native="true">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>60</y>
        <width>651</width>
        <height>531</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>400</width>
        <height>400</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">QWidget#widget{
	
	border-image: url(:/telainicial/telainicial/tec.png);
border-radius:10px;

text-transform: uppercase;
}
QLineEdit{
border-radius:0px;
background-color:rgba(0, 0, 0, 0);
border:1px solid rgba(0, 0, 0, 0);
border-bottom-color:rgba(46, 82, 101, 255);
color:white;
padding-bottom:7px;
text-transform: uppercase;
font: 14pt &quot;MS Shell Dlg 2&quot;;
}


QPushButton{
background-color:rgba(2, 65, 118, 255);
color:rgba(255, 255, 255, 200);
border-radius:5px;
text-transform: uppercase;
}
QPushButton:pressed{
padding-left:5px;
padding-top:5px;
background-color:rgba(2, 65, 118, 100);
background-position:calc(100% - 10px)center;
}
QPushButton:hover{
background-color:rgba(2, 65, 118, 200);
}</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QLabel" name="label">
         <property name="styleSheet">
          <string notr="true">font: 75 20pt &quot;MS Shell Dlg 2&quot;;
color: rgb(255, 255, 255);
text-transform: uppercase;</string>
         </property>
         <property name="text">
          <string>seja bem vindo</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QLabel" name="lb_log">
         <property name="sizeIncrement">
          <size>
           <width>0</width>
           <height>40</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">font: 18pt &quot;MS Shell Dlg 2&quot;;
background-color:none;
color: rgb(244, 244, 244);</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="tx_login">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>40</height>
          </size>
         </property>
         <property name="sizeIncrement">
          <size>
           <width>0</width>
           <height>40</height>
          </size>
         </property>
         <property name="baseSize">
          <size>
           <width>0</width>
           <height>40</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <property name="placeholderText">
          <string>login</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="tx_senha">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>40</height>
          </size>
         </property>
         <property name="contextMenuPolicy">
          <enum>Qt::ActionsContextMenu</enum>
         </property>
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <property name="echoMode">
          <enum>QLineEdit::Password</enum>
         </property>
         <property name="placeholderText">
          <string>senha</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QFrame" name="frame">
         <property name="styleSheet">
          <string notr="true">QFrame{border:none;
	background-color:none;
	background-image: none;
}</string>
         </property>
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <property name="spacing">
           <number>9</number>
          </property>
          <property name="leftMargin">
           <number>9</number>
          </property>
          <property name="topMargin">
           <number>9</number>
          </property>
          <property name="rightMargin">
           <number>9</number>
          </property>
          <property name="bottomMargin">
           <number>9</number>
          </property>
          <item>
           <widget class="QPushButton" name="bt_logar">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>40</height>
             </size>
            </property>
            <property name="font">
             <font>
              <pointsize>18</pointsize>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">QPushButton#pushButton{
background-color:rgba(2, 65, 118, 255);
color:rgba(255, 255, 255, 200);
border-radius:5px;
}
QPushButton#pushButton:pressed{
padding-left:5px;
padding-top:5px;
background-color:rgba(2, 65, 118, 100);
background-position:calc(100% - 10px)center;
}
QPushButton#pushButton:hover{
background-color:rgba(2, 65, 118, 200);
}</string>
            </property>
            <property name="text">
             <string>login</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="bt_canelar">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>40</height>
             </size>
            </property>
            <property name="font">
             <font>
              <pointsize>18</pointsize>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true"/>
            </property>
            <property name="text">
             <string>cancelar</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QFrame" name="frame_2">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>10</y>
        <width>101</width>
        <height>101</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>0</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(55, 111, 167);
border-radius:50%;
background-image: url(:/telainicial/telainicial/cadastro clientes.png);
 height: 200px;
  background-position: center; /* Center the image */
  background-repeat: no-repeat; /* Do not repeat the image */
 </string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
     </widget>
    </widget>
   </item>
  </layout>
  <action name="actionsenha">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../assets/file.qrc">
     <normaloff>:/telainicial/telainicial/olho.png</normaloff>:/telainicial/telainicial/olho.png</iconset>
   </property>
   <property name="text">
    <string>senha</string>
   </property>
   <property name="toolTip">
    <string>VER SENHA</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="../assets/file.qrc"/>
 </resources>
 <connections/>
</ui>
